stl/inc/xpolymorphic_allocator.h

```diff
@@ -282,6 +282,18 @@ namespace pmr {
             return _Resource;
         }
 
+        _NODISCARD_FRIEND bool operator==(

```

Comment:
Given that this is not in the actual C++20 standard, I would prefer to see a reference to `LWG-3683` here.

Reply:
It seems that MSVC STL tends to reference LWG issues in standard headers only if their resolution haven't been accepted. When the issue is officially resolved, STL tends to remove the related comments.

Reply:
mmh, okay. I don't like this policy but given that it is policy, I'm good with it.